### ISSUE TITLE
CI: Test Failure SingleNodeDiscoveryIT on Windows 

### ISSUE THREAD

**Issue by tbrooks8** [elastic/elasticsearch#23902]
There are multiple Windows builds failing on SingleNodeDiscoveryIT:

http://build-us-00.elastic.co/job/es_core_master_windows-2012-r2/5206/#showFailuresLink
http://build-us-00.elastic.co/job/es_core_master_windows-2012-r2/5207/#showFailuresLink

2> REPRODUCE WITH: gradle :core:integTest -Dtests.seed=89B8A7295A4D83B9 -Dtests.class=org.elasticsearch.discovery.single.SingleNodeDiscoveryIT -Dtests.nightly=false -Dtests.method="testSingleNodesDoNotDiscoverEachOther" -Dtests.security.manager=true -Dtests.nightly=false -Dtests.heap.size=1011m -Dtests.jvm.argline="-server -XX:+UseConcMarkSweepGC -XX:-UseCompressedOops" -Dtests.locale=ca -Dtests.timezone=Europe/Belgrade

```ERROR   0.64s J1 | SingleNodeDiscoveryIT.testSingleNodesDoNotDiscoverEachOther <<< FAILURES!
   > Throwable #1: java.lang.RuntimeException: failed to start nodes
   > 	at __randomizedtesting.SeedInfo.seed([89B8A7295A4D83B9:4E29264E28E0A2B]:0)
   > 	at org.elasticsearch.test.InternalTestCluster.startAndPublishNodesAndClients(InternalTestCluster.java:1372)
   > 	at org.elasticsearch.test.InternalTestCluster.reset(InternalTestCluster.java:1051)
   > 	at org.elasticsearch.test.InternalTestCluster.beforeTest(InternalTestCluster.java:978)
   > 	at org.elasticsearch.discovery.single.SingleNodeDiscoveryIT.testSingleNodesDoNotDiscoverEachOther(SingleNodeDiscoveryIT.java:164)
   > 	at java.lang.Thread.run(Thread.java:745)
   > Caused by: java.util.concurrent.ExecutionException: BindTransportException[Failed to bind to [49152-49156]]; nested: BindException[Address already in use: JVM_Bind];
   > 	at java.util.concurrent.FutureTask.report(FutureTask.java:122)
   > 	at java.util.concurrent.FutureTask.get(FutureTask.java:192)
   > 	at org.elasticsearch.test.InternalTestCluster.startAndPublishNodesAndClients(InternalTestCluster.java:1367)
   > 	... 40 more
   > Caused by: BindTransportException[Failed to bind to [49152-49156]]; nested: BindException[Address already in use: JVM_Bind];
   > 	at org.elasticsearch.transport.TcpTransport.bindToPort(TcpTransport.java:736)
   > 	at org.elasticsearch.transport.TcpTransport.bindServer(TcpTransport.java:701)
   > 	at org.elasticsearch.transport.MockTcpTransport.doStart(MockTcpTransport.java:406)
   > 	at ```

**Comment by jaymode** [comment]
Another failure: http://build-us-00.elastic.co/job/es_core_master_windows-2012-r2/5213/console

**Comment by jasontedor** [comment]
Closed by 24127bf41686e04eae9930aafad374c004b99b74
